dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java

```diff
@@ -155,18 +155,18 @@ public GenericData parse(RoutedMessage routedMessage) {
         genData.addPair("ibm_methodName", logRecord.getSourceMethodName());
         genData.addPair("ibm_className", logRecord.getSourceClassName());
 
-        KeyValuePairList extensions = new KeyValuePairList();
         Map<String, String> extMap = null;
```

Comment:
minor comment - should move declaration of extMap down to line below where it is set.  keeps code neater as then you don't need to think whether extMap is used in code later when you are reading the code.